src/doc/python_scripting/functions.rst

```diff
@@ -1610,37 +1611,15 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineMaterialExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
-    The DefineMaterialExpression function creates new material variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
-
-
-**Example:**
-
-::
-
-  #% visit -cli
-  OpenDatabase("/usr/gapps/visit/data/globe.silo")
-  DefineScalarExpression("myvar", "sin(u) + cos(w)")
-  # Plot the scalar expression variable.
-  AddPlot("Pseudocolor", "myvar")
-  DrawPlots()
-  # Plot a vector expression variable.
-  DefineVectorExpression("myvec", "{u,v,w}")
-  AddPlot("Vector", "myvec")
-  DrawPlots()
+    DefineMaterialExpression creates new material variables.
```

Comment:
I am not sure of the merit of documenting a function that doesn't actually do anything. Shouldn't we just remove this from the CLI?

Reply:
I started a discussion on this. Perhaps we should provide a way to create a material variable.